Makefile

```diff
@@ -204,45 +213,47 @@ build-libwaku: $(LIBWAKU)
 test-libwaku: | $(LIBWAKU)
 	go test -tags '$(BUILD_TAGS) use_nwaku' -run TestDial ./wakuv2/... -count 1 -v -json | jq -r '.Output'
 
+LIBSDS := $(CURDIR)/vendor/github.com/waku-org/sds-go-bindings/third_party/nim-sds/build/libsds.$(LIBSDS_EXT)
```

Comment:
This implies that we vendor dependencies and breaks my efforts made for https://github.com/status-im/status-go/issues/6970.

SDS library should be provided externally, like I did with `nwaku` in
- https://github.com/status-im/status-go/pull/7026

With only difference that instead of providing source path (`NWAKU_SOURCE_DIR` in my pr), you might want to provide binaries path (`NIM_SDS_HEADER_PATH` and `NIM_SDS_LIB_PATH` in your pr).